src/components/Primary-Button/Button.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { DEFAULT_PROPS, SIZE_CONFIGS } from './Button.constants';
+import { StyledButton, ButtonContent, LoadingSpinner } from './Button.styles';
+
+import type { ButtonProps } from './Button.types';
+
+export const PrimaryButton: React.FC<ButtonProps> = ({
+  variant = DEFAULT_PROPS.variant, // variant tiene tipo ButtonVariant
```

Comment:
There are inline comments in Spanish here. Project rule requires code and comments in English only. Translate comments to English so the repository remains consistent for all contributors.

**Sugerencia:**
```typescript
Replace Spanish comments with English equivalents, e.g.:
// variant is typed as ButtonVariant
// Ensure size is one of 'small' | 'medium' | 'large'
// Use fontSizeRem to compute spinner size
```